recipe/meta.yaml

```diff
@@ -31,29 +30,34 @@ requirements:
     - wheel
     - llvmdev 11.1.*
```

Comment:
To check with numba if we should be building with llvm 11 or llvm 14.
https://github.com/numba/llvmlite/blob/v0.40.0/CHANGE_LOG#LL16C4-L16C4

Reply:
Looks like numba requires llvm 14 https://github.com/conda-forge/llvmlite-feedstock/pull/69#issuecomment-1533229442 but checking the upstream source would be nice to do

Reply:
dunno, on Readme it says:

<html>
<body>
<!--StartFragment-->

llvmlite versions | compatible LLVM versions
-- | --
0.37.0 - ... | 11.x.x

<!--EndFragment-->
</body>
</html>

Reply:
The maintainers have forgotten to update docs for mo than 2 years https://github.com/numba/llvmlite/commit/a8513785563703f3edc20e5d420e10b346e0586a